react/react1/week3/to-do-app/src/Todo.css

```diff
@@ -0,0 +1,158 @@
+body {
```

Comment:
this should be in App.css

Reply:
sorry, @27ManishaPatel I wasn't clear, I meant the whole code.

Reply:
Hi can you say it again that what is not clear I am not getting  you . As I have updated with some changes what you have suggested and some are still undone about to create separate component .

Reply:
oh, you dismissed the first comment, so you can't see
but in Todo.css I meant you can move the whole code to App.css